library/src/main/java/me/mattlogan/library/ViewStack.java

```diff
@@ -3,58 +3,82 @@
 import android.animation.Animator;
 import android.animation.AnimatorListenerAdapter;
 import android.os.Bundle;
+import android.support.annotation.LayoutRes;
+import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
 import java.util.ArrayList;
 import java.util.EmptyStackException;
 import java.util.List;
-import java.util.Stack;
 
 import static me.mattlogan.library.Preconditions.checkNotNull;
 import static me.mattlogan.library.Preconditions.checkStringNotEmpty;
 
 /**
- * This manages a navigation stack by representing each item in the stack as a ViewFactory, which is
+ * This manages a navigation stack by representing each item in the stack as a layout id, which is
  * responsible for View creation. All standard Java Stack operations are supported, with additional
  * methods for pushing and popping with animated transitions.
  */
 public final class ViewStack {
 
-    private final Stack<ViewFactory> stack = new Stack<>();
+    public static final int SINGLE_VIEW = -1;
+    private final ParcelableIntStack stack = new ParcelableIntStack();
     private final ViewGroup container;
     private final ViewStackDelegate delegate;
     private final List<StackChangedListener> listeners = new ArrayList<>();
+    private final PancakesViewInflator pancakesViewInflator;
```

Comment:
Can we just name the instance `viewInflator` internally? I don't think keeping the pancakes prefix around does us any good.